projects/storefrontlib/src/cms-components/checkout/components/shipping-address/shipping-address.component.ts

```diff
@@ -23,77 +27,131 @@ export interface CardWithAddress {
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class ShippingAddressComponent implements OnInit {
-  existingAddresses$: Observable<Address[]>;
-  newAddressFormManuallyOpened = false;
-  isLoading$: Observable<boolean>;
-  cards$: Observable<CardWithAddress[]>;
-  selectedAddress$: Observable<Address>;
+  paymentType = '';
+  addressFormOpened = false;
```

Comment:
Will there be a way to handle a var name change when we address breaking changes?

Reply:
we can do auto update in `schematics` for renamed variables.